packages/react-instantsearch-hooks-dom/src/widgets/__testutils__/all-widgets.tsx

```diff
@@ -56,6 +56,9 @@ function Widget<TWidget extends SingleWidget>({
     case 'RefinementList': {
       return <widget.Component attribute="brand" {...props} />;
     }
+    case 'SearchBox': {
+      return <widget.Component onSubmit={undefined} {...props} />;
```

Comment:
I wonder if something is wrong with the props that makes this onSubmit required to be passed in a way compatible with both div and form, we should look into this after merging